packages/medusa/src/models/swap.ts

```diff
@@ -137,6 +138,9 @@ export class Swap {
   @Column({ type: "boolean", nullable: true })
   no_notification: Boolean
 
+  @Column({ type: "boolean", default: false })
+  allow_backorder: Boolean
+
```

Comment:
Generally, we don't want to allow backorder on swaps, hence why we default to `false`